docs/administration-guide/20-custom-maps.md

```diff
@@ -0,0 +1,25 @@
+## Custom Maps
+
+By default, Metabase uses OpenStreetMaps for map visualizations, but there are a few customization options.
+
+### Finding Map Settings
+
+To find the map settings:
+
+1. Click the gear icon in the upper right.
+2. Select `Maps` from the navigation on the left.
+
+![Map Settings](images/MapSettings.png)
+
+### Map Tile Server
+
+If you'd like to use a map tile server other than OpenStreetMaps, simply add the map tile server URL to the first field on the page.
```

Comment:
Should we quickly explain why you would want to use a different map tile server?